src/azure-cli/HISTORY.rst

```diff
@@ -188,6 +188,10 @@ Release History
 * Add preview param `--replicas-per-master` to `az redis create` command
 * Update azure-mgmt-redis from 6.0.0 to 7.0.0rc1
 
+**Security**
```

Comment:
please rebase to dev branch and make your release note in next release.

Reply:
Agreed. Let's put it under the latest version - 2.0.81.

Reply:
I have edited the history file.